web/app/routes/authenticated/document.js

```diff
@@ -103,10 +103,8 @@ export default class DocumentRoute extends Route {
       console.log("Error recording analytics: " + err);
     }
 
-    // If not a draft, record the doc as recently viewed.
-    if (!params.draft) {
-      this.recentDocs.addDoc.perform(params.document_id);
-    }
+    // Record the doc with the RecentlyViewedDocs service.
+    void this.recentDocs.markViewed.perform(params.document_id, params.draft);
```

Comment:
Should we add some error checking here? Curious about the use of `void` as well.

Reply:
Another place where I'm relying on `markViewed`'s error handlers. And because I'm not awaiting this task, I'm not sure what error handling I could do. But let me know your thoughts.

I use `void` on async functions (that return void) to signal when I'm _deliberately not awaiting them_. Here I just want to kick off the task and not have it hold up the `model` call. I'm hoping `void` reminds me of this the next time I'm in the code. Shouldn't have any affect on the function when used this way.

Reply:
Okay, makes sense 👍